data_architect_misc/data_transformer/transform_functions/apac_singapore_transform_functions.py

```diff
@@ -0,0 +1,79 @@
+"""This is the subclass of Transform function for Singapore (APAC division).
+
+We will define transform functions specific to Singapore here.
+
+Author: Jholman Jaramillo
+Last Modified: September 03, 2020
+"""
+
+import re
+import os
+import pandas as pd
+from glob import glob
+
+import transform_errors
+from constants import comp_harm_constants
+from constants.transform_constants import KEY_CURRENT_INPUT_FILE, KEY_DELIMITER, KEY_HEADER
+from transform_functions.common_comp_harm_transform_functions import CommonCompHarmTransformFunctions
+from qa_functions.common_comp_harm_qa_functions import CommonCompHarmQAFunctions
+
+
+class ApacSingaporeTransformFunctions(CommonCompHarmTransformFunctions):
+    """
+    All custom (uncommon) transform functions **SPECIFIC to
+    individual processing task** must be defined as part
+    of this class.
+    """
+    SINGAPORE_SPECIFIC_BRAND_MAPPINGS = {
```

Comment:
Add a comment ('#') to explain why we map these brands to these advertisers in SG.